packages/fluentui/react-northstar/src/components/Button/Button.tsx

```diff
@@ -232,7 +232,6 @@ export const Button = compose<'button', ButtonProps, ButtonStylesProps, {}, {}>(
         {...rtlTextContainer.getAttributes({ forElements: [children] })}
         {...getA11yProps('root', {
           onClick: handleClick,
-          disabled,
```

Comment:
Removing this line because it break the condition

` disabled: props.disabled && !props.loading ? (props.as === 'button' ? true : undefined) : undefined,`

which is in ButtonBehavior.ts file.